stories/spectra/mass.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Meta } from '@storybook/react';
+
+import { Axis, LineSeries, Plot } from '../../src';
+import data from '../data/infrared.json';
+import { DEFAULT_CONFIG } from '../utils';
+
+export default {
+  title: 'Experimental spectra/Infrared',
```

Comment:
Isn't it mass?

Reply:
Thanks. A mistake I often do, copy paste a file and change the original file rather than the copy.